docs/source/conf.py

```diff
@@ -107,7 +108,9 @@
 
 # General information about the project.
 project = u'Apache Arrow'
-copyright = u'2016-2019 Apache Software Foundation'
+# Prevent oddities with a weird clock
+year = max(2021, datetime.datetime.now().year)
```

Comment:
Why do we need this?
Can we assume that we ensure setting correct clock on machine that is used for generating documentations?

If we hard-code `2021` here, we need to update this next year.

Reply:
Ah, maybe I was being overly cautious - I've changed it to just assume the year is correct.